keyboards/meetlab/kafkasplit/readme.md

```diff
@@ -0,0 +1,26 @@
+# kafkasplit - PCB
+
+![kafka68](https://i.imgur.com/fqxZGeHh.jpg)
+
+A 44-key split keyboard with rgb
+
+* Keyboard Maintainer: https://github.com/LXF-YZP
+* Hardware Supported: Lucky PCB
```

Comment:
Different PCB? It would also be helpful to provide some info about the hardware, like a vendor link or Github repository.

Reply:
I have open source the keyboard pcb and keyboard cover and put the github link in the readme.md file. We will continue to improve the open source files in the future.